examples/layout/base/lib/main.dart

```diff
@@ -29,3 +30,4 @@ class MyApp extends StatelessWidget {
     );
   }
 }
+// #enddocregion all
```

Comment:
Can you add this new line back and regenerate the diff? It seemed to add a ["\ No newline at end of file"](https://github.com/flutter/website/blob/755235eace7e63aff1a19346af174d14366c5790/src/development/ui/layout/tutorial.md?plain=1#L194) to the diff.

```suggestion
// #enddocregion all

```